src/modules/variable/init.rs

```diff
@@ -11,13 +11,18 @@ pub struct VariableInit {
     name: String,
     expr: Box<Expr>,
     global_id: Option<usize>,
-    is_fun_ctx: bool
+    is_fun_ctx: bool,
+    is_const: bool
```

Comment:
Please add a trailing comma here as well, for reasons given previously; and check for other places in this PR where it is possible to do so.